packages/idyll-cli/bin/cmds/create-project.js

```diff
@@ -123,6 +129,9 @@ async function createProject(answers) {
   console.log(colors.success(startMessage));
   var spinner;
   try {
+    if (!isinstallDependencies) {
+      stages = stages.filter(stage => stage[0] !== 'Installing dependencies');
```

Comment:
This will break if we change the text of the message to something other than `'Installing dependencies'`. I think a slightly cleaner approach would be to only ever add this stage if the flag is set to `true`, rather than removing it if the flag is `false`.

Reply:
agreed !